src/auditor/instruction_set.jl

```diff
@@ -36,12 +36,14 @@ function instruction_mnemonics(path::AbstractString, platform::Platform)
     output = IOBuffer()
 
     # Run objdump to disassemble the input binary
-    if platform isa MacOS || platform isa FreeBSD
+    if Sys.isbsd(platform)
         objdump_cmd = "llvm-objdump -d $(basename(path))"
     else
         objdump_cmd = "\${target}-objdump -d $(basename(path))"
     end
+    @info("Fetching mnemonics for $(path)")
     run_interactive(ur, `/bin/bash -c "$(objdump_cmd)"`; stdout=output, stderr=devnull)
+    @info("done!", readdir(dirname(path)))
```

Comment:
Remember to remove this before merging

Reply:
I was just editing that, heh.  I guess we're both watching the logs. :P